tests/formats/bzng/write.yaml

```diff
@@ -0,0 +1,49 @@
+zql: '*'
+
+input: |
+  #0:record[_path:string,ts:time,d:float64]
+  0:[a;10;1;]
+  0:[xyz;20;1.5;]
+
+output-format: bzng
+
+output: |
```

Comment:
I think the format for encoding binary here is really neat. Would you consider using a different tag, instead of the `HEX` marker? For example, `outputHex` or something similar; it would be an error to use both `output` and `outputHex`.